.clang-format

```diff
@@ -0,0 +1,11 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+ColumnLimit: 110
+IndentWidth: 4
+AccessModifierOffset: -4
+SortIncludes: false # reordering may break existing code
+ConstructorInitializerIndentWidth: 8
+ContinuationIndentWidth: 8
+
+
```

Comment:
I think you could remove one of the new line.

Reply:
Done